src/locales/en.json

```diff
@@ -72,11 +72,13 @@
   "Internal Shopify Order ID": "Internal Shopify Order ID",
   "Instance Url": "Instance Url",
   "Inventory": "Inventory",
+  "Inventory cost": "Inventory cost",
   "Job details": "Job details",
   "Job Manager": "Job Manager",
-  "Loading": "Loading",
+  "Landed inventory cost": "Landed inventory cost",
```

Comment:
we don't need to save these as UI labels since they will be populated dynamically from job enum

Reply:
Yes sir! Done